src/javascript/app_2/App/Components/Form/Datepicker/datepicker.jsx

```diff
@@ -0,0 +1,202 @@
+import classNames      from 'classnames';
```

Comment:
Better to follow the previous filename `date_picker.jsx` as we have the same for `time_picker.jsx`. (Also the folder to `DatePicker`)